include/swift/Option/FrontendOptions.td

```diff
@@ -939,6 +939,11 @@ def cxx_interop_disable_requirement_at_import :
   HelpText<"Do not require C++ interoperability to be enabled when importing a Swift module that enables C++ interoperability">,
   Flags<[FrontendOption, HelpHidden]>;
 
+def cxx_interop_libcxx_path: JoinedOrSeparate<["-"], "clang-libcxx-path">,
```

Comment:
Why is this spelt with a `-clang` prefix? I think that we shouldn't use `libcxx` because this is confusing as people struggle to differentiate:

- libcxx
- libc++
- libstdc++
- stl_port
- msvcprt

Can we make this generic like `-cxx-stl-path`? I think that using `-stdlib=` to match clang might be confusing.

Reply:
Agreed that we shouldn't call this `libcxx`, however, Clang doesn't use `stl` in the user-facing command like flags, so I wouldn't use it for Swift either. I would probably call this `-clang-cxx-stdlib=` or `-clang-stdlib=` or `-cxx-stdlib=`.

Reply:
Renaming to `-cxx-stdlib-path`.